documentation/src/docs/asciidoc/release-notes/release-notes-5.9.0-M1.adoc

```diff
@@ -26,6 +26,8 @@ on GitHub.
 * `ConfigurationParameters` has now a `keySet()` method allowing to retrieve
   all configuration parameter keys.
 * New `IterationSelector` for selecting a subset of a test's or container's iterations.
+* `ParallelExecutionConfiguration` has a new method allowing to supply a predicate to
+   be called when the ForkJoinPool is saturated.
```

Comment:
```suggestion
   be called when the `ForkJoinPool` is saturated.
```

Reply:
After having re-read the documentation, it's my understanding that the predicate is actually used to decide if the pool should be allowed "to saturate".

In other words, I don't think the predicate is called when the pool is already "saturated".

Reply:
I interpreted "saturated" as meaning the number of threads in the pool has reached the maximum and they are all busy or blocked. In this case, the predicate is used to allow it to be saturated (true) or not (false, in which case an exception is thrown.) But I'm happy to rephrase it if you think it makes more sense.